post-scan-actions/fss-to-securityhub/lambda/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "lambda",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
While checking in node_modules is not a good practice, we should provide a npm script to install dependencies and zip the whole folder. So the readme should note a npm prerequisite and the instructions to run the script.